### ISSUE TITLE
Not capturing SteamID when logging into panel

### ISSUE THREAD

**Issue by ResqDiver1317** [CADOJRP/FiveM-AdministrationPanel#34]
I tried setting up the latest from the Master Branch as well as the latest release (0.4.1) and I'm getting the same issue with both. When I login to the panel, it shows me a sample dashboard, but no options to do anything regarding creating a community or adding a server or even editing the panel. I look in my database, and the user it created shows blanks for name and steamid. I set them both manually, and then am able to setup the server and edit the panel. When I try to warn, kick, or ban, it works, but does not save a name associated with the action to reflect who issues the action. I was able to test with 1 warning from in game, and it went through properly and logged in the user's record and included who did it, but further attempts to do any warnings/etc... in game return zero results at all. 

I went ahead and stuck with the 0.4.1 version due to all the issues being reported about bans not working and the E1001 errors on the latest master branch release. Any advice that can be offered would be greatly appreciated!

**Comment by avery1227** [comment]
Is your Steam API key correct?

Also E1001 is caused by CRON job failing.

**Comment by ResqDiver1317** [comment]
It should be correct. I saw someone else with a similar complaint in the discord so I cancelled my key and issued a new one and updated my config file. I'll redo it and give that a shot. Out of curiosity, with the Steam API key, where it asks for the address, does it need to be specific directly to the panel, or is the domain (or IP in my case) good enough?

And I'll cuss at the CRON more as well. I thought I had it correct, and when I manually enter the address, it doesn't show any issues, but it's possible it's just not running on timer like it should due to something I set wrong. 

currently I was using 
* * * * * http://MYSERVERIP/html/lsdojrpPanel/api/cron

**Comment by ResqDiver1317** [comment]
I've messed with the steamID as much as I can and no luck. Not sure why, but it is still not logging my steamID in the users table unless I do it manually, and even then, when a 2nd person connects to the panel, it does not add a new record to the DB users table

**Comment by ResqDiver1317** [comment]
I think there was something about my hosting that was not playing nice with this. I did notice that my MySQL version (5.5.62) did not support the way the database was trying to setup so I updated the MySQL on my server and was able to build the DB properly, but it still wasn't capturing name or steamid when I would sign into it. I got frustrated and ended up moving the ELHosting and it instantly started working so something about my dedicated server's setup wasn't playing nice. Perhaps the install wiki should be updated to reflect minimum server specs / php/mysql versions.
